frontend/taipy-gui/src/components/Taipy/Menu.tsx

```diff
@@ -38,6 +38,7 @@ const baseTitleProps = { noWrap: true, variant: "h6" } as const;
 const Menu = (props: MenuProps) => {
     const { label, onAction = "", lov, width, inactiveIds = emptyArray, active = true } = props;
     const [selectedValue, setSelectedValue] = useState<string | string[]>("");
+    const [currentSelection, setCurrentSelection] = useState<string>("");
```

Comment:
I don't think we need a separate selection management

Reply:
Removed in the latest commit